docs/configuration/modules/loadflow-validation.md

```diff
@@ -27,10 +27,9 @@ both ones. The default value of this property is `false`.
 The `epsilon-x` property is an optional property that defines the value used to correct the reactance in flows validation.
 The default value of this property is `0.1`.
 
-## load-flow-factory
-The `load-flow-factory` property is an optional property that defines the `com.powsybl.loadflow.LoadFlowFactory` implementation
-to use for running the loadflow. If this property is not set, the value of the `LoadFlowFactory` property of the
-[componentDefaultConfig](componentDefaultConfig.md) module is used.
+## load-flow-name
+The `load-flow-name` property is an optional property that defines the implementation name to use for running the loadflow. 
+If this property is not set, the default loadflow implementation is used.
```

Comment:
Same remark here

Reply:
added link to configuration/modules/load-flow.md